utilities/test_suite/HOST/Tensor_misc_host.cpp

```diff
@@ -178,7 +200,11 @@ int main(int argc, char **argv)
                     fill_mean_stddev_values(nDim, maxSize, meanTensor, stdDevTensor, qaMode, axisMask, scriptPath);
 
                 startWallTime = omp_get_wtime();
-                rppt_normalize_host(inputF32, srcDescriptorPtrND, outputF32, dstDescriptorPtrND, axisMask, meanTensor, stdDevTensor, computeMeanStddev, scale, shift, roiTensor, handle);
+
+                if(bitDepth == 0)
+                    rppt_normalize_host(inputU8, srcDescriptorPtrND, outputU8, dstDescriptorPtrND, axisMask, meanTensor, stdDevTensor, computeMeanStddev, scale, shift, roiTensor, handle);
```

Comment:
Why the u8 calls are addded here . you have added the same calls in both audio and image test suites. Please remove the changes for u8 if it is not needed